src/commands/kv/mod.rs

```diff
@@ -72,6 +72,7 @@ fn api_client(user: &GlobalUser) -> Result<HttpApiClient, failure::Error> {
             // This is useful for bulk upload operations.
             http_timeout: Duration::from_secs(5 * 60),
         },
+        Environment::Production,
```

Comment:
i don't understand this change.

Reply:
this is part of the new cloudflare-rs version :)

Reply:
but what does it do?

Reply:
it uses api.cloudflare.com instead of staging.api.cloudflare.com--the cloudflare-rs team introduced a new staging environment you can use to test your code on staging API endpoints with this client library.

Reply:
cool! i'll ask more questions about it over there.